app/renderer/js/utils/translation-util.ts

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+import path = require('path');
+import electron = require('electron');
+import i18n = require('i18n');
+
+let instance: TranslationUtil = null;
+let app: Electron.App = null;
+
+/* To make the util runnable in both main and renderer process */
+if (process.type === 'renderer') {
+	app = electron.remote.app;
+} else {
+	app = electron.app;
+}
+
+class TranslationUtil {
+	constructor() {
+		if (instance) {
+			return this;
+		}
+
+		instance = this;
+		i18n.configure({
+			directory: path.join(__dirname, '../../../translations/'),
+			register: this
+		});
+	}
+
+	__(phrase: string): string {
+		return i18n.__({ phrase, locale: app.getLocale() });
```

Comment:
Should we also set `ENGLISH` here as a fallback?

Reply:
I doubt that it's required because `app.getLocale()` should always work, but it's never bad to be safe, so I added a commit for that anyway. We can drop that if others feel it is unnecessary.

Reply:
I'm assuming you're talking about the corner case where `app.getLocale()` returns a falsey value.

Reply:
Correct.